### DISCUSSION TITLE
总是显示重定向到smartdns失败

### DISCUSSION THREAD

**Discussion by makeye** [pymumu/smartdns#1139] [Q&A]
不管端口怎么改变，其他选项在没设置插件一直显示重定向失败。检查过其他设置也没问题（先是能正常重定向，解析叶比较快，然后过一会酒自己弹出重定向失败，这个时候就开始抽风了，解析特别慢），唯一一个办法就是重定向到53端口，但是速度就特别慢，希望作者能够告诉我解决办法，谢谢

**Comment by pymumu** [comment]
用的什么版本？

重定向失败，一般就是有其他软件修改了配置，导致dns失效了。
要排查你安装的软件列表。

**Reply by makeye** [reply]
用的是1.2022.36.1这个版本的，如果端口使用默认到6053和5335而且不开重定向到话，能正常用，但是速度就没啥用了，其实其他端口也能用（所有端口开重定向就失败了），哥，能告诉我解决怎样设置吗？或者搞个电报群大家一起讨论交流嘛

**Reply by pymumu** [reply]
测速和你用哪个端口没有关系。
如果能用6053，就用6053也是一样的。

有些固件软件上会和smartdns冲突的。

**Reply by pymumu** [reply]
可以发一下/tmp/etc/smartdns/smartdns.conf看看内容

**Reply by makeye** [reply]
这是配置文件
    ![IMG_20221028_132309.jpg](https://user-images.githubusercontent.com/28622643/198509983-ae5bd5a9-2b91-4b7e-9119-88c24f48a089.jpg)

**Reply by PikuZheng** [reply]
> 用的是1.2022.36.1这个版本的

版本不对啊，dnsmasq重定向需要至少38.0.10
另外主程序版本和luci版本号要一致

**Reply by makeye** [reply]
这是那个coolshare宝哥他们搞的一个机子，也就是ARS2这个机子，smartdns也是从他应用商店下的
